learning_journal/routes.py

```diff
@@ -8,4 +8,4 @@ def includeme(config):
     config.add_route("home", "/")
     config.add_route("edit", "/edit")
```

Comment:
You should probably make the `edit` route dependent upon the ID of a given journal entry.